cmd/olympus/actions/actions_test.go

```diff
@@ -8,34 +8,28 @@ import (
 	"github.com/gobuffalo/gocraft-work-adapter"
 	"github.com/gobuffalo/suite"
 	"github.com/gocraft/work"
-	"github.com/gomods/athens/pkg/config/env"
+	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/eventlog"
-	"github.com/gomods/athens/pkg/eventlog/mongo"
 	"github.com/gomods/athens/pkg/payloads"
-	"github.com/gomods/athens/pkg/storage/mem"
+)
+
+const (
+	testConfigFile = "../../../config.test.toml"
 )
 
 type ActionSuite struct {
 	*suite.Action
 }
 
 func Test_ActionSuite(t *testing.T) {
-	stg, err := mem.NewStorage()
-	if err != nil {
-		t.Fatalf("error creating storage (%s)", err)
+	conf := config.GetConfLogErr(testConfigFile, t)
+	if conf == nil {
```

Comment:
Do we need that? Wouldn't GetConfLogErr fail the test on error? I'm also not sure if we should put a func used only in tests (GetConfLogErr) in the config pkg.

Reply:
I wouldn't mind little repetition in this case tbh - getConf + err check

Reply:
oh sorry - I completely glossed over the first part of your comment. No, you're right - there's no need for that err check. I'll fix that